gems/quilt_rails/app/controllers/quilt/performance_report_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Quilt
+  class PerformanceReportController < ActionController::Base
+    protect_from_forgery except: :index
```

Comment:
The bot is mad at me, but we don't really care about forgery protection here since it does not actually access data at all. I tried using `null_session` here but it was still yelling at me about csrf tokens, so this seemed fine.

Reply:
Might be something to pass by security?

Reply:
This is a POST endpoint, it parses a JSON body.

Reply:
@clayton-shopify could you take another look quick? I got `with: :null_session` working now